### DISCUSSION TITLE
Place script at the end of body

### DISCUSSION THREAD

**Discussion by vadim-su** [DioxusLabs/dioxus#3324] [Q&A] [answered]
Is it possible to place a script at the end of body?
Currently this code place it in the dioxus div wrapper
![image](https://github.com/user-attachments/assets/b0f35864-719d-458a-a40b-0d8b0f0df7e6)
![image](https://github.com/user-attachments/assets/334375d7-97c5-4df7-b171-be8a3df2a8fc)

**Comment by ealmloff** [comment]
The `document::Script` component inserts scripts in the head

**Reply by vadim-su** [reply]
But how can I place the script at the end of the body?

**Reply by ealmloff** [reply, marked as answer]
You could do that with javascript in `eval` that manually manipulates the DOM. rsx is only rendered in the head or inside the main div

There is some documentation about breaking out of dioxus' rendering model [here](https://dioxuslabs.com/learn/0.6/essentials/breaking/#synchronizing-dom-updates-with)
